dotcom-rendering/src/web/components/FilterKeyEventsToggle.importable.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/react';
+import { from, neutral, remSpace } from '@guardian/source-foundations';
+import { ToggleSwitch } from '@guardian/source-react-components-development-kitchen';
+import { useEffect, useState } from 'react';
+
+const cssOverrides = css`
+	padding: ${remSpace[2]};
+
+	${from.phablet} {
+		padding-left: ${remSpace[5]};
+	}
+
+	${from.desktop} {
+		border-top: 1px solid ${neutral[86]};
+		padding-left: 0;
+	}
+`;
+
+export const FilterKeyEventsToggle = () => {
+	const [checked, setChecked] = useState(false);
+
+	const handleClick = () => {
+		setChecked(!checked);
+
+		const urlParams = new URLSearchParams(window.location.search);
+		urlParams.set('filterKeyEvents', checked ? 'false' : 'true');
+
+		window.location.search = urlParams.toString();
+		window.location.hash = 'maincontent';
```

Comment:
may be that we're already doing this but we should make sure to keep the toggle in view at the top of the screen when navigating to the main content after a click - would be weird if the button disappears after you click it

Reply:
Resolved